common/scripts/ci/run_boiler_app_integration_tests_with_live_backend.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -i
+set -ex
+
+: "${HOST:?}"
+: "${SDK_BACKEND:?}"
+: "${SDK_VERSION:?}"
+if [[ "$SDK_BACKEND" == 'BEAR' ]]; then
+    : "${USER_NAME:?}"
+fi
+
+export BOILER_APP_NAME=new-boiler-app-$SDK_LANG
+export BOILER_APP_VERSION=app-toolkit@$SDK_VERSION
+export BOILER_APP_HOST=https://127.0.0.1:8080
+CYPRESS_IMAGE='cypress/included:12.10.0'
```

Comment:
Please use cypress images from ECR

Reply:
do later on ticket unify cypress for all repo